ZenPacks/zenoss/LinuxMonitor/zenpack.yaml

```diff
@@ -103,15 +121,32 @@ classes:
                 grid_display: false
                 api_only: true
                 api_backendtype: method
+            filesystem:
+                label: File System
+                api_only: true
+                api_backendtype: method
+                type: entity
+            mountpoint:
+                grid_display: False
+                details_display: False
+                index_type: field
+            major_minor:
+                label: "Major:Minor"
+                grid_display: False
+
 
     SnapshotVolume:
         label: Snapshot Volume
-        impacted_by:
-            - volumeGroup
-            - logicalVolume
-        impacts:
-            - openstack_core_components
+        dynamicview_views: [service_view]
+        dynamicview_relations:
+            impacted_by:
+                - volumeGroup
```

Comment:
No need to have SnapshotVolume impacted by its volumeGroup as that impact will come via its logicalVolume.